### PR TITLE
Split gradle_plugin_test.dart

### PR DESCRIPTION
**Pull request by blasten**
## Description

Split `gradle_plugin_test.dart`, since this test started timing out.

## Tests

I added the following tests:

n/a

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Review by dnfield — APPROVED** [review]
LGTM

File: dev/devicelab/bin/tasks/gradle_plugin_bundle_test.dart [hunk 1]
```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_devicelab/framework/framework.dart';
```
Comment (zanderso) [inline dev/devicelab/bin/tasks/gradle_plugin_bundle_test.dart]: Alphabetize imports

File: dev/devicelab/bin/tasks/gradle_plugin_bundle_test.dart [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import 'package:flutter_devicelab/framework/apk_utils.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+
+String javaHome;
```
Comment (zanderso) [inline dev/devicelab/bin/tasks/gradle_plugin_bundle_test.dart]: Do these have to be globals? Can they at least go under main()?

**Review by zanderso — APPROVED** [review]
lgtm w/ a tiny bit of cleanup.

File: dev/devicelab/bin/tasks/gradle_plugin_bundle_test.dart (hunk 2 above)
Comment (blasten) [reply dev/devicelab/bin/tasks/gradle_plugin_bundle_test.dart]: I cleaned this up, so no need for javaHome here :)

**Pull request merged**